test/packages/bad_fields/data_stream/foo/fields/fields.yml

```diff
@@ -5,3 +5,10 @@
   date_format: yyyy-MM-dd
 - name: object_without_object_type.*
   type: object
+- name: object_with_subfields
+  description: This field should be of type group, because it contains subfields.
+  type: object
+  object_type: keyword
```

Comment:
`object_type` is not necessary here, right?

Reply:
Yeah, it wouldn't be neccesary to produce this validation error, but I added it so it doesn't trigger also the error about missing `object_type` for `object`.